### ISSUE TITLE
File upload/download shouldn't use absolute paths

### ISSUE THREAD

**Issue by 23Skidoo** [genomizer/genomizer-server#167]
For obvious security reasons, e.g. `download?path=/etc/passwd`. There should be a configurable data directory + possibly per-user subdirs.

**Comment by 23Skidoo** [comment]
Looks like it touches a lot of database code, so maybe the database group should look into it.

**Comment by 23Skidoo** [comment]
Right now we use a configurable upload directory (`fileLocation` in the settings file) + per-experiment directories.

**Comment by 23Skidoo** [comment]
Haven't managed to implement this before the code freeze; most recent patch is on the [`relative-paths`](https://github.com/genomizer/genomizer-server/tree/relative-paths) branch.

**Comment by azoimide** [comment]
This is done?

**Comment by 23Skidoo** [comment]
No, this is not done. Needs testing and maybe further work. I'd prefer not to merge right now.

**Comment by 23Skidoo** [comment]
Related: #431.
